packages/focusmanager/src/react/use-focus-manager.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unnecessary-type-assertion */
```

Comment:
This is a weird line to have to disable eslint for this rule.  Where *is* the type assertion here?

Reply:
needed for refs that need to be typed like focus

Reply:
oh, I see.  It's a disable for the whole file. I was reading it as a next-line like most of the others. 👍